akka-stream/src/main/scala/akka/stream/javadsl/Compression.scala

```diff
@@ -34,6 +34,16 @@ object Compression {
     scaladsl.Compression.gzip.asJava
 
   /**
+    * Same as [[gzip]] but with a custom level. The level can be:
```

Comment:
Actually, the value can be anything between 0 and 9. See the [Javadoc](https://docs.oracle.com/javase/8/docs/api/java/util/zip/Deflater.html#Deflater-int-). It's basically the usual gzip/deflate compression levels.

Reply:
:+1: